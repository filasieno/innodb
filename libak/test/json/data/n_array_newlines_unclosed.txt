name=n_array_newlines_unclosed
max_depth=32
max_string_size=2048
max_json_size=1048576
----------
["a",
4
,1,