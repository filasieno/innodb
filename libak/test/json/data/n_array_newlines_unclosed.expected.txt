---
STATE_CHANGED_EVENT: STATE_INITIALIZED
BEGIN_ARRAY
STRING_VALUE "a" more=0
INT 4
INT 1
---
---
PARSE_EOF_EVENT
STATE_CHANGED_EVENT: STATE_DONE
